conbench/tests/app/test_conceptual_benchmark_views.py

```diff
@@ -9,6 +12,34 @@ def assert_response_is_login_page(resp):
     assert '<label for="password">Password</label>' in resp.text, resp.text
 
 
+benchmark_result_dict = {
+    "run_id": "1",
+    "batch_id": "1",
+    "timestamp": "2020-11-25T21:02:44Z",
+    "context": {},
+    # something like "machine_info": {"foo": "bar"} should work, relax
+    # constraints.
+    "machine_info": _fixtures.MACHINE_INFO,
+    # This is special repo/commit data that makes the web application use mock
+    # commit data (instead of reaching out to the GitHub HTTP API). The goal
+    # here is to use a commit that looks like it's on 'the default branch',
+    # because only such results are entering the BMRT cache as of now.
+    "github": {
+        "commit": "4beb514d071c9beec69b8917b5265e77ade22fb3",
+        "repository": "https://github.com/org/repo",
+    },
+    "stats": {
+        "data": [
+            "1.1",
+        ],
+        "unit": "s",
+    },
+    "tags": {
+        "name": "fun-benchmark",
+    },
+}
```

Comment:
off-topic, side note.

With less properties being required such a dictionary becomes more and more expressive/explicit on its own. 

I envision a state where `batch_id`, `run_id`, `context` are not required here, too.

Minimal set of properties for a benchmark result to start to make sense:
- name
- hardware/platform
- data
- time